pkg/api/v20240812preview/openshiftcluster_validatestatic.go

```diff
@@ -408,3 +423,56 @@ func (sv openShiftClusterStaticValidator) validateDelta(oc, current *OpenShiftCl
 
 	return nil
 }
+
+func (sv openShiftClusterStaticValidator) validatePlatformWorkloadIdentityProfile(path string, pwip *PlatformWorkloadIdentityProfile) error {
+	// PlatformWorkloadIdentityProfile being empty is acceptable
+	if pwip == nil {
+		return nil
+	}
+
+	// Validate the PlatformWorkloadIdentities
+	for n, p := range pwip.PlatformWorkloadIdentities {
+		if !stringutils.IsResourceIDFormatted(p.ResourceID) {
+			return api.NewCloudError(http.StatusBadRequest, api.CloudErrorCodeInvalidParameter, fmt.Sprintf("%s.PlatformWorkloadIdentities[%d].resourceID", path, n), "ResourceID %s formatted incorrectly.", p.ResourceID)
+		}
+
+		if stringutils.ResourceType(p.ResourceID) != "identities" {
```

Comment:
I believe this should be `userAssignedIdentities` instead of `identities`. If you want to look at an example of the type of managed identity that will be used for the platform workloads, you can do an [`az identity show`](https://learn.microsoft.com/en-us/cli/azure/identity?view=azure-cli-latest#az-identity-show) against one of the AKS managed identities in our dev subscription.

Reply:
Thanks! Fixed